src/main/java/pl/simpleascoding/tutoringplatform/api/AdvertisementController.java

```diff
@@ -0,0 +1,27 @@
+package pl.simpleascoding.tutoringplatform.api;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.simpleascoding.tutoringplatform.dto.AdvertisementDTO;
+import pl.simpleascoding.tutoringplatform.dto.CreateAdvertisementDTO;
+import pl.simpleascoding.tutoringplatform.dto.RscpDTO;
+import pl.simpleascoding.tutoringplatform.service.advertisement.AdvertisementService;
+
+@RestController
+@RequestMapping("api/v1/advertisement")
+@RequiredArgsConstructor
+public class AdvertisementController {
+
+    private final AdvertisementService advertisementService;
+
+    @PostMapping("/add")
```

Comment:
We need to decide on how our endpoints should be named. We'll have to discuss this on Discord, so for now leave it as is.